openapi/spec.json

```diff
@@ -1177,6 +1183,168 @@
         }
       }
     },
+    "/devices/{organizationId}/{deviceId}/routes/{routeId}/activate": {
```

Comment:
What's the returned status if we activate an already activated route?

Reply:
"404 Not Found"

Reply:
![image](https://user-images.githubusercontent.com/19814268/51763315-95cd6e00-20d2-11e9-9e6c-88eef5e122f9.png)
I will add this to the doc